src/scale.c

```diff
@@ -4,25 +4,10 @@
 #include "avif/internal.h"
 
 #if !defined(AVIF_LIBYUV_ENABLED)
-
-avifBool avifImageScale(avifImage * image,
-                        uint32_t dstWidth,
-                        uint32_t dstHeight,
-                        uint32_t imageSizeLimit,
-                        uint32_t imageDimensionLimit,
-                        avifDiagnostics * diag)
-{
-    (void)image;
-    (void)dstWidth;
-    (void)dstHeight;
-    (void)imageSizeLimit;
-    (void)imageDimensionLimit;
-    avifDiagnosticsPrintf(diag, "avifImageScale() called, but is unimplemented without libyuv!");
-    return AVIF_FALSE;
-}
-
+#include "scale/avif_scale.h"
+#define ScalePlane_12 avifScalePlane_12
+#define ScalePlane avifScalePlane
```

Comment:
I would rather use a `#if defined(AVIF_LIBYUV_ENABLED)` below than these hard-coded aliases.
If we keep `ScalePlane()` and `ScalePlane_12()` exactly as they are in libyuv, I think it is fine to keep their original names. No need to prepend `avif`.